### ISSUE TITLE
Occasional ambiguity warnings?

### ISSUE THREAD

**Issue by IainNZ** [JuliaOpt/JuMP.jl#383]
Has anyone else seen non-deterministic ambiguity warnings?

**Comment by joehuchette** [comment]
Are you on 0.3?

**Comment by IainNZ** [comment]
Yeah 0.3.3

**Comment by mlubin** [comment]
Haven't seen this here.

**Comment by joehuchette** [comment]
I haven't seen any, but I usually use 0.4. Do you often use some other (unrelated) package at the same time?

**Comment by IainNZ** [comment]
Its quite possible. The fact you've never seen suggests... something. I'll reopen if I can recreate, I've always been distracted when its happening.

**Comment by IainNZ** [comment]
```
Warning: New definition
    *(S<:Union(Variable,GenericAffExpr{Float64,Variable},GenericQuadExpr{Float64,Variable}),T<:GenericQuadExpr{Float64,Variable}) at /Users/idunning/.julia/v0.3/JuMP/src/operators.jl:266
is ambiguous with:
    *(T<:GenericQuadExpr{Float64,Variable},S<:Union(Variable,GenericAffExpr{Float64,Variable},GenericQuadExpr{Float64,Variable})) at /Users/idunning/.julia/v0.3/JuMP/src/operators.jl:264.
To fix, define
    *(T<:GenericQuadExpr{Float64,Variable},T<:GenericQuadExpr{Float64,Variable})
before the new definition.
```

**Comment by IainNZ** [comment]
Stochastic though - JuMPeR always involed.

**Comment by mlubin** [comment]
Well it seems like the warning is correct.

**Comment by IainNZ** [comment]
I just got this in something not involving JuMPeR at all, btw. Am I the only one who ever sees it?!

**Comment by yeesian** [comment]
I've seen it before, and just saw it again:
```
Warning: New definition 
    *(S<:Union(Variable,GenericQuadExpr{Float64,Variable},GenericAffExpr{Float64,Variable}),T<:GenericQuadExpr{Float64,Variable}) at /Users/yeesian/.julia/v0.3/JuMP/src/operators.jl:275
is ambiguous with: 
    *(T<:GenericQuadExpr{Float64,Variable},S<:Union(Variable,GenericQuadExpr{Float64,Variable},GenericAffExpr{Float64,Variable})) at /Users/yeesian/.julia/v0.3/JuMP/src/operators.jl:273.
To fix, define 
    *(T<:GenericQuadExpr{Float64,Variable},T<:GenericQuadExpr{Float64,Variable})
before the new definition.
```

**Comment by mlubin** [comment]
Let's just fix the ambiguity then?

**Comment by yeesian** [comment]
closed via #438?

**Comment by mlubin** [comment]
Let's reopen this if we see it again
